### PR TITLE
Responsible parties

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

File: src/client/app/responsible-party/responsible-party-group.component.ts [hunk 1]
```diff
@@ -46,24 +51,39 @@ export class ResponsiblePartyGroupComponent extends AbstractGroupComponent<Respo
...
+            this.setItemsCollection(siteLogModel[this.partyType.getObjectName()]);
+            this.init();
+        }
+    }
+
+    @Input()
+    set originalSiteLogModel(originalSiteLogModel: any) {
+        if (originalSiteLogModel) {
+            this.setItemsOriginalCollection(originalSiteLogModel[this.partyType.getObjectName()]);
+        }
```
Comment (lbodor) [inline src/client/app/responsible-party/responsible-party-group.component.ts]: The two inputs above (siteLogModel and originalSiteLogModel) have been moved to AbstractGroupComponent earlier today. Could they stay there?

**Comment by lbodor** [comment]
:+1:

File: src/client/app/responsible-party/responsible-party-group.component.ts (hunk 1 above)
Comment (frankfu03) [inline src/client/app/responsible-party/responsible-party-group.component.ts]: Yeah, the two inputs have been removed from responsible-party-group.component.ts.

But we have to keep them in site-identification and site-location components as the two components do not extend from AbstractGroupComponent.

**Comment by frankfu03** [comment]
Yeah, the two inputs have been removed from responsible-party-group.component.ts.

But we have to keep them in site-identification and site-location components as the two components do not extend from AbstractGroupComponent.

**Pull request merged**